src/test/spec/runner/test_file.rs

```diff
@@ -37,8 +39,8 @@ pub struct RunOn {
 impl RunOn {
     pub fn can_run_on(&self, client: &EventClient) -> bool {
         if let Some(ref min_version) = self.min_server_version {
-            let req = VersionReq::parse(&format!(">= {}", &min_version)).unwrap();
-            if !req.matches(&client.server_version) {
+            let req = VersionReq::parse(&format!("< {}", &min_version)).unwrap();
```

Comment:
Quirk of using `VersionReq`: a release candidate version is not considered to be greater than a lower non-release candidate version (e.g. `4.4.0-rc11` is not >= `4.2`). Flipping the logic here fixes that issue.

Reply:
This is linked in the source code for `semver` -- the logic makes sense but doesn't seem as applicable to the driver. Is there any reason we'd need to know if a server version is prerelease? If not, maybe it would be better to strip out the tag to avoid accidentally skipping the latest server version in future tests. https://docs.npmjs.com/misc/semver#prerelease-tags

Reply:
From grepping the specifications repo for `rc0` through `rc10`, there doesn't seem to be any usage of it in any JSON or YAML files. There are a few things about `listCollections` with certain rc versions in the spec itself, but those are for versions below 3.6, so we don't need to worry about them. There's also a mention of an rc in the auth spec, but since it's not in any of the tests, we don't need to worry about it here. Based on all that, I think it's probably fine to strip it out for now. We can revisit if there ever ends up being any spec tests that use prelease tags.